test/lean/errors.expected.lean

```diff
@@ -35,11 +35,7 @@ abbrev SailM := PreSailM RegisterType trivialChoiceSource exception
 
 XXXXXXXXX
 
-import Out.Sail.Sail
-import Out.Sail.BitVec
-import Out.Sail.IntRange
-import Out.Defs
-import Out.Specialization
+import Out.String
```

Comment:
Is it the case that the expected files do not contain the submodules? it would be better to have everything no?

Reply:
It's usually considered better style to "shake" imports in Lean projects, Mathlib even has a linter for it

Reply:
I meant that AFAIU the `Out.String` module is generated from sail, so it should be in the `expected.lean` file

Reply:
Ah, yea, that needs to be fixed in the testing infrastructure